src/code/client.coffee

```diff
@@ -229,6 +230,12 @@ class CloudFileManagerClient
             return @alert(err) if err
             @_fileOpened content, metadata, {openedContent: content.clone()}, @_getHashParams metadata
 
+  openUrlFile: (url) ->
+    urlProvider = new URLProvider()
```

Comment:
I realized it could be used again, if the app were calling `openUrlFile` from the API. I just instantiated it at the top. (It's not part of `@allProviders` because that's the app-selected list, and we want this to always be there.)